session/session.go

```diff
@@ -312,32 +313,32 @@ func sanitizeBalInfo(balInfo perun.BalInfo) {
 	}
 }
 
-// retrieveParts retrieves the peers from corresponding to the aliases from the contacts provider.
+// retrieveParts retrieves the peer IDs corresponding to the aliases from the ID provider.
 // The order of entries for parts list will be same as that of aliases. i.e aliases[i] = parts[i].Alias.
-func retrieveParts(aliases []string, contacts perun.ContactsReader) ([]perun.Peer, error) {
-	knownParts := make(map[string]perun.Peer, len(aliases))
-	parts := make([]perun.Peer, len(aliases))
+func retrieveParts(aliases []string, idProvider perun.IDReader) ([]perun.PeerID, error) {
+	knownParts := make(map[string]perun.PeerID, len(aliases))
+	parts := make([]perun.PeerID, len(aliases))
 	missingParts := make([]string, 0, len(aliases))
 	repeatedParts := make([]string, 0, len(aliases))
```

Comment:
```suggestion
	knownPartIDs := make(map[string]perun.PeerID, len(aliases))
	partIDs := make([]perun.PeerID, len(aliases))
	missingPartIDs := make([]string, 0, len(aliases))
	repeatedPartIDs := make([]string, 0, len(aliases))
```

As we renamed `Peer` to `PeerID`, renaming these variables accordingly.
Please update the variable names in other places of usage as well.

Reply:
kept parts itself.